### ISSUE TITLE
Accessibility bug: Focus not visible on links on page

### ISSUE THREAD

**Issue by lindalu-MSFT** [SharePoint/sp-dev-docs#4065]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
Users who rely on keyboard navigation will be affected if the focus indicator is not properly visible.

#### Observed Behavior
Focus indicator is not properly visible for SharePoint link, On page navigation links,  Contributors link, This page button.

And also some part of focus indicator is visible for the previous link when focus is on some other links.

#### Steps to Reproduce
Repro Steps:​
Step 1: Go to [https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/use-fabric-react-components](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/use-fabric-react-components)
Step 2: Navigate to the page using tab key.
Step 3: Verify that focus indicator is not properly visible for the links.

Original bug reported here: https://office.visualstudio.com/OC/_queries/edit/3294372/?triage=true

**Comment by andrewconnell** [comment]
The VS.com link you shared isn't publically accessible. However, I'm not able to repro this in multiple browsers... specifically for the "contributor" links, I see the focus indicator (as well as everywhere else) as I tab through the page...

![image](https://user-images.githubusercontent.com/2068657/58701684-8eb0b800-8371-11e9-853d-6ab050715edd.png)

**Comment by lindalu-MSFT** [comment]
@andrewconnell Thank you for verifying that this is now working. I closed our bug out as Resolved/External. I appreciate your prompt reply.
